src/cli/restore/sharepoint.go

```diff
@@ -76,6 +78,18 @@ func addSharePointCommands(cmd *cobra.Command) *cobra.Command {
 			utils.ListItemFN, nil,
 			"Restore list items by ID")
 
+		fs.StringSliceVar(
+			&pageFolders,
+			utils.PageFN, nil,
+			"Restore Site pages by SharePoint Site ID",
+		)
+
+		fs.StringSliceVar(
+			&pages,
+			utils.PageItemFN, nil,
```

Comment:
The flag name should get updated to match the other formats.  Ie: the flag for page folders should be `page-folders`, and items should be `pages`.  Less confusion that way.

Reply:
Resolved with [5414c71](https://github.com/alcionai/corso/pull/2228/commits/5414c71581c3cd12499e4eb6cf934bbd78464a1f)